ddcommon-ffi/cbindgen.toml

```diff
@@ -0,0 +1,51 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+language = "C"
+tab_width = 2
+header = """// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+"""
+include_guard = "DDOG_COMMON_H"
+style = "both"
+
+no_includes = true
+sys_includes = ["stdbool.h", "stddef.h", "stdint.h"]
+
+after_includes = """
+
+#if defined(_MSC_VER)
+#define DDOG_CHARSLICE_C(string) \\
+/* NOTE: Compilation fails if you pass in a char* instead of a literal */ {.ptr = "" string, .len = sizeof(string) - 1}
+#else
+#define DDOG_CHARSLICE_C(string) \\
+/* NOTE: Compilation fails if you pass in a char* instead of a literal */ ((ddog_CharSlice){ .ptr = "" string, .len = sizeof(string) - 1 })
+#endif
+
+#if defined(__cplusplus) && (__cplusplus >= 201703L)
+#  define DD_CHECK_RETURN [[nodiscard]]
+#elif defined(_Check_return_) /* SAL */
+#  define DD_CHECK_RETURN _Check_return_
+#elif (defined(__has_attribute) && __has_attribute(warn_unused_result)) || \\
+      (defined(__GNUC__) && (__GNUC__ >= 4))
+#  define DD_CHECK_RETURN __attribute__((__warn_unused_result__))
+#else
+#  define DD_CHECK_RETURN
```

Comment:
Should the prefix on these macros be `DDOG_`, for consistency?

Reply:
PROBABLY.

Reply:
Will change